### PR TITLE
fix: correct broken links

### PR DESCRIPTION
**Pull request by lahabana**
Signed-off-by: Charly Molter <charly.molter@konghq.com>

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -31,6 +31,64 @@ const dirTree = require('directory-tree')
 const path = require("path");
 const fs = require("fs");
 const versions = require("./versions.js");
+const installMethods = [
```
Comment (parkanzky) [inline docs/.vuepress/config.js:34]: Is this all related to broken links checker not exiting correctly?

**Review by parkanzky — APPROVED** [review]
LGTM

File: docs/.vuepress/config.js (hunk 1 above)
Comment (lahabana) [reply docs/.vuepress/config.js:34]: Nope it's just cleanup from a weirdly placed file

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -22,8 +22,14 @@ jobs:
...
-          max_timeout: 1200
+          max_timeout: 2400
       - name: link checker
+        timeout-minutes: 30
         run: |
           npm install broken-link-checker -g
-          blc $URL -rof --exclude 127.0.0.1 --exclude 192.168 --exclude redis.io  | grep -v '───OK───'
+          if [[ ! `blc $URL/docs/dev -rof --exclude 127.0.0.1 --exclude 192.168 --exclude redis.io > out` ]]; then
+            cat out | grep -v  '───OK───'
+            exit 1
```
Comment (michaelbeaumont) [inline .github/workflows/ci.yml:32]: What am I missing, isn't the first line irrelevant if we always `exit 1`? What went wrong with the original line?

Comment (lahabana) [reply .github/workflows/ci.yml:32]: the `$?` is the exit code of grep not of the cmd so this was always succeeding

Comment (parkanzky) [reply .github/workflows/ci.yml:32]: You could use $PIPESTATUS[0] to get the exit code of the first command in the pipeline.

Comment (parkanzky) [reply .github/workflows/ci.yml:32]: Although, looks like $PIPESTATUS is pretty bash-specific. Probably best to keep it compatible with other shells.

Comment (michaelbeaumont) [reply .github/workflows/ci.yml:32]: Where was `$?` checked? I thought `pipefail` was set with GH actions.

**Comment by lahabana** [comment]
Can't manage to make the job actually fail. I'm going to revert the change in the github action so we at least fix the links

**Pull request merged**